### PR TITLE
csv upload tweaks etc.

### PR DESCRIPTION
**Pull request by fisx**
- text encoding issues.
- if csv files contain empty lines, those should not trigger bogus user creation.
- ~~tweak avatar image browser caching.~~
- tweak avatar image size selection.
- devops niceties.

### CODE REVIEW

**Comment by fisx** [comment]
Caching of avatar breaks test suite.  We could play with the 'applicationCacheEnabled' capability in hs-webdriver, but I guess the failure means something.  Some perfromance problems (due to bad wlan) have been reported in practice, so I'm not sure how to proceed here.  I'll ask poldi!  Let's see how the performance turns out in practice.

File: .travis/docker-build.sh [hunk 1]
```diff
@@ -3,6 +3,8 @@
 /liqd/aula/.travis/docker-link-stack-workdir.sh
 /liqd/aula/docker/selenium.sh restart
 
+apt-get install libicu-dev --yes
```
Comment (andorp) [inline .travis/docker-build.sh]: This should be part of the aula-docker dockerfile. But it is ok for now.

**Comment by andorp** [comment]
unsafePermormIO should be avoided, we can fix that in a separate PR. LGTM

File: .travis/docker-build.sh (hunk 1 above)
Comment (fisx) [inline .travis/docker-build.sh]: I agree, of course; it was just easier to do it here.

**Comment by fisx** [comment]
I'm curious: I half-reasoned out that in this case `unsafePerformIO` is not a problem.  Of course it's always better to not have to do the reasoning, but have you thought of anything specific that can go wrong here if we don't refactor it away?

Anyway, thanks for the review, well-spotted.  I'll merge, then.

**Pull request merged**